apps/Node.js/install

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-version=v19.0.0
+version=v19.4.0
 version2=v0.38.0
```

Comment:
@theofficialgman, the auto-updater should have updated version2 to v0.39.3. Could you check why it did not happen?

Reply:
@Botspot Look up at the message from this PR. It tells you why it didn't. The link failed the validate url check

Reply:
also lol, this is great
```
webVer[0]=$(wget -qO- https://nodejs.org/en/download/current/ | grep 'https://nodejs\.org/dist/.*\.tar\.gz' -m1 | tr '/' '\n' | grep '^v')
armhf_url[0]="https://nodejs.org/dist/${webVer}/node-${webVer}-linux-armv7l.tar.xz"
arm64_url[0]="https://nodejs.org/dist/${webVer}/node-${webVer}-linux-arm64.tar.xz"

webVer[1]=$(get_release nvm-sh/nvm)
all_url[1]="https://raw.githubusercontent.com/nvm-sh/nvm/${webVer[1]}/install.sh"
```
I never thought I using it that was but it is totally valid and working syntax for my script 
great work there

Reply:
What exactly was unique with my implementation here? It looks fairly similar to Cura's script.

Reply:
you used a separate armhf and arm64 url for the first index in the array and then an all_url for the second index in the array. as you know, the array index determines what number gets appended to the `version` variable. index 0 is `version`, index 1 is `version2`, index 2 is `version3`, etc

the cura script uses armhf and arm64 urls for both version and version2. you use armhf and arm64 for version and one all url for version2.

the script is written well so it works but I just never intended or even thought of that use case.